code/modules/cargo/exports/large_objects.dm

```diff
@@ -40,27 +40,18 @@
 	export_types = list(/obj/structure/closet/crate/coffin)
 
 /datum/export/large/reagent_dispenser
-	cost = 100 // +0-400 depending on amount of reagents left
```

Comment:
wtf man this wasnt me. Dont blame legit everything on me please?

Reply:
I was only blaming you for the fermenting barrels. The reagent dispensers were like this for a long time now, sorry for the misunderstanding.